app/src/main/java/com/example/keybindhelper/Database.java

```diff
@@ -6,7 +6,7 @@
 import androidx.room.RoomDatabase;
 import androidx.sqlite.db.SupportSQLiteOpenHelper;
 
-public class AppDatatabase extends RoomDatabase{
+public class Database extends RoomDatabase{
```

Comment:
Changed class name from AppDatatabase to just Database because there was a typo in the name and another file exists in the dao package called AppDatabase.

Reply:
- "Database" is ambiguous.  Is there a more specific name you can give this class?  What is being stored in the database?